modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+.Procedure
+Use the following procedure to deploy {product-title} for vSphere storage on different zones and regions.
+
+. In the VMware vCenter vSphere client GUI:
+.. Create and assign `openshift-zone` and `openshift-region` tags as needed to the hosts on the desired clusters.
```

Comment:
openshift-zone & openshift-regions are categories not tags.

Reply:
Done.